<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupbox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>395</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Raster definitions</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_possibility">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Possibility raster</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QgsMapLayerComboBox" name="maplayer_possibility"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_necessity">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Necessity raster</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QgsMapLayerComboBox" name="maplayer_necessity"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
